app/main/views/marketplace.py

```diff
@@ -41,10 +43,15 @@ def index():
             "framework {} status {}".format(framework.get('slug'), framework.get('status')))
         abort(500)
 
+    template_data = {
+        'frameworks': {framework['slug']: framework for framework in frameworks},
+        'temporary_message': temporary_message
+    }
+    if logged_in:
+        template_data['logged_in'] = True
     return render_template(
         'index.html',
-        frameworks={framework['slug']: framework for framework in frameworks},
-        temporary_message=temporary_message
+        **template_data
```

Comment:
I think this approach is wrong actually. We used it before when we used to send a standard dict into each template for asset paths, etc...

I'll amend it to the existing pattern.